### ISSUE TITLE
The problem with downloading jar file

### ISSUE THREAD

**Issue by ghost** [mpeltonen/sbt-idea#278]
When I run sbt gen-idea, the building process hangs on the line [info] downloading http://repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.10.2/scala-reflect-2.10.2-javadoc.jar ...

No warnings, no errors, just hanging and does nothing else. The file scala-reflect-2.10.2-javadoc.jar exists on the specified path and is loaded by the browser, but not loaded sbt.

Here is build.sbt:

```
name := "MyProject"

version := "0.1"

scalaVersion := "2.10.3"

retrieveManaged := true

resolvers += "spray" at "http://repo.spray.io/"

resolvers += "repo.codahale.com" at "http://repo.codahale.com"

libraryDependencies ++= Seq(
  "org.scalatest" % "scalatest_2.10" % "2.0" % "test",
  "com.dongxiguo" %% "fastring" % "0.2.2",
  "io.spray" %%  "spray-json" % "1.2.5",
  "javax.servlet" % "servlet-api" % "2.4",
  "org.eclipse.jetty" % "jetty-server" % "9.1.1.v20140108",
  "mysql" % "mysql-connector-java" % "5.1.12"
)
```

**Comment by jozic** [comment]
i've just tried it and it works fine (though I probably have this jar downloaded)
 does it hang every time you run it?

**Comment by ghost** [comment]
Yes, it happens every time, i can't generate the project structure for Idea. Maybe it is possible to download the file manually and add its local file system path to build.sbt, but I do not know what i must add to build.sbt for it. Sorry, i'm Scala and sbt newbie.

upd: I spent half a day with this problem, but now worked correctly. Magic.

**Comment by jozic** [comment]
so, does it work now?

**Comment by ghost** [comment]
Yes, it works and I did not understand what was going on.

**Comment by jozic** [comment]
glad it works now
I suggest to close the issue until we can reproduce it 100%
